fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Utils.java

```diff
@@ -429,4 +431,78 @@ public static boolean canOnlyDoBroadcast(PhysicalHashJoinOperator node,
         }
         return Optional.empty();
     }
+
+    private static boolean isIntegerOrDecimalType(Type t) {
```

Comment:
```suggestion
    private static boolean isFixedNumericType(Type t) {
```

Add this method to Type class

Reply:
There has a samliar method `Type.isExactNumericType()`